lib/engine/data_test.go

```diff
@@ -7,7 +7,7 @@ import (
 	"github.com/globalsign/mgo/bson"
 )
 
-var update = flag.Bool("update", false, "Update the expected test values in golden file") // please keep this line until https://github.com/kubernetes-sigs/service-catalog/issues/2319#issuecomment-425200065 is fixed
+var _ = flag.Bool("update", false, "Update the expected test values in golden file") // please keep this line until https://github.com/kubernetes-sigs/service-catalog/issues/2319#issuecomment-425200065 is fixed
```

Comment:
Il me semble que tu pourrais carrément te passer de l'affectation dans ce cas là
```suggestion
flag.Bool("update", false, "Update the expected test values in golden file") // please keep this line until https://github.com/kubernetes-sigs/service-catalog/issues/2319#issuecomment-425200065 is fixed
```
À vérifier que ça réveille pas le beugue.

Reply:
Effectivement ! Je vais essayer !

Reply:
Apparemment le linter n'aime pas cette idée:

![image](https://user-images.githubusercontent.com/531781/115203423-b80a9680-a0f7-11eb-8a6b-c5586190b609.png)

Je suppose que les seuls appels de fonctions tolérés en top-level sont destinés à populer une variable.